cli/tests/unit/http_test.ts

```diff
@@ -1355,6 +1355,66 @@ Deno.test({
   },
 });
 
+Deno.test({
+  name: "http server compresses streaming body - check body",
```

Comment:
```suggestion
  name: "http server compresses streamed body (direct resource fast path) - check body",
```

And move down to the other "compresses streaming body" tests.